### PR TITLE
v2.0.1

### PR DESCRIPTION
**Pull request by leewang0**
## Description of the change

- Update README with examples of usage for each of the categories in this sdk
- Add ATS.ApplicationsApi `applicationsChangeStageCreate` method, which updates a candidate's application from one stage to another (i.e. phone screen -> onsite)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests, or the description above explains how testing was performed

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or gif attached. Download "Kap" from the Mac App store for easy gif screen capture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,95 @@ The documentation for various Merge category API's can be found here:
 * [HRIS](https://www.merge.dev/docs/hris/overview/) (Human Resource Information Systems)
 * [Ticketing](https://www.merge.dev/docs/ticketing/overview/)
 
+### Basic Usage
+
+Common to all categories is the Merge sdk `Configuration` object, which defines the authentication properties of 
+your requests. The `apiKey` property is the api key of your Merge account, which can be administered in the 
+[Merge dashboard](https://app.merge.dev/configuration/keys). The `accessToken` property will be the 
+`X-Account-Token` header which identifies which of your customers' data you are requesting. See the following
+examples for calling various category api's:
```
Comment (eltonkl) [inline README.md]: nit: `SDK`, `API`, `APIs`

File: README.md [hunk 2]
```diff
@@ -23,6 +23,95 @@ The documentation for various Merge category API's can be found here:
...
+
+#### Accounting
+
+```
+let conf_accounting = new Configuration({
```
Comment (eltonkl) [inline README.md]: nit: `const` over `let` - immutable by default is good but this is example code so nbd

less nitty - can you use camelCase for these variable names? it's more consistent with most JS in the SDK and what i've seen in the wild

File: README.md (hunk 1 above)
Comment (leewang0) [reply README.md]: good eye

File: README.md [hunk 3]
```diff
@@ -23,6 +23,95 @@ The documentation for various Merge category API's can be found here:
...
+```
+const confAccounting = new Configuration({
+    apiKey: "REDACTED-YourAPIKeyWithMerge",
+    accessToken: "REDACTED-YourCustomer1Key"
+});
+
+const acctApi = new merge_sdk.Accounting.AccountsApi(confAccounting);
+
+// lists the accounting accounts for customer 1
+let response = await acctApi.accountsList({}).catch((reason) => {
```
Comment (eltonkl) [inline README.md]: nit pt 2: `const`s for these `let`s

and another nit: skip the abbreviations lol, imo makes it harder to read

**Review by eltonkl — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (leewang0) [reply README.md]: eh i think i'll leave the let on the response but you're right on the variable naming, shouldn't cut corners.

**Pull request merged**